tools/.gitignore

```diff
@@ -6,3 +6,4 @@ pokemon_animation
 pokemon_animation_graphics
 gfx
 stadium
+make_patch
```

Comment:
Sort these alphabetically, and make sure there's a newline at the end of the file (no red "(-)" icon in the GitHub diff).

Reply:
Sorted in https://github.com/pret/pokecrystal/pull/882/commits/2e74ae318e2f04ee7ba7d37bc050b81413c11afb